### PR TITLE
CreateTag and CreateEdge support TTL

### PR DESCRIPTION
**Pull request by haoxins**
## What type of PR is this?
- [ ] bug
- [x] feature
- [ ] enhancement

### CODE REVIEW

File: label.go [hunk 1]
```diff
@@ -19,21 +28,23 @@ type Label struct {
...
+	Name        string
+	Fields      []LabelFieldSchema
+	TTLDuration uint
+	TTLCol      string
+}
+
 type LabelFieldSchema struct {
 	Field    string
 	Type     string
 	Nullable bool
```
Comment (Nicole00) [inline label.go:41]: add defaut value

Comment (haoxins) [reply label.go:41]: can we add this in another PR?
may need some tests and changes for this (`default` supports)

Comment (Nicole00) [reply label.go:41]: Of course you can.

Comment (haoxins) [reply label.go:41]: I created a issue for this 
https://github.com/vesoft-inc/nebula-go/issues/330

**Review by Nicole00 — APPROVED** [review]

**Pull request merged**